### DISCUSSION TITLE
Save Row Into External Table & Internal Table

### DISCUSSION THREAD

**Discussion by syeillah** [Budibase/budibase#4654] [Help]
Hi, so i want to save data form by separate them into 2 tables. **Non attachment** field will be on external, and special for **attachment** field will be save into internal DB. 

eg:
field A => save row into external table 1
field B => save row into external table 1
field C => save row into internal table 2

<img width="923" alt="Screen Shot 2022-02-23 at 15 26 00" src="https://user-images.githubusercontent.com/99949174/155284199-046adff1-63bd-4e7c-bd5f-0bb81e3f551c.png">

Any solutions for this? 
Thank you!

**Comment by mjashanks** [comment]
Hey @syeillah - you should keep everything in one form, and have 2 `Save Row` actions use the fields on the form to save to separate tables.

I detail this in here: https://github.com/Budibase/budibase/discussions/4461#discussioncomment-2204142
